pkgs/applications/networking/jdownloader2/default.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv, lib, fetchurl, writeScript, makeDesktopItem, copyDesktopItems, jre, imagemagick, coreutils }:
+
+let
+
+  icon = fetchurl {
+    url = "https://jdownloader.org/_media/vote/trazo.png";
+    sha256 = "3ebab992e7dd04ffcb6c30fee1a7e2b43f3537cb2b22124b30325d25bffdac29";
+  };
+
+  wrapper = writeScript "jdownloader" ''
+    #! ${stdenv.shell}
+    PATH=${lib.makeBinPath [ jre coreutils ]}
+    JDJAR=''${XDG_DATA_HOME:-$HOME/.local/share}/jdownloader/JDownloader.jar
+    dir=`dirname "$0"`
+    if [ ! -f ''${JDJAR} ]; then
+        install -Dm755 $dir/JDownloader.jar ''${JDJAR}
+    fi
+    ${jre}/bin/java -jar ''${JDJAR} "''${@}"
+  '';
+
+in stdenv.mkDerivation rec {
+  pname = "jdownloader2";
+  version = "2.0";
+
+  src = fetchurl {
+    url = "https://installer.jdownloader.org/JDownloader.jar";
```

Comment:
Please upload it to web.archive.org.

Reply:
I had to restore http://, https:// does not work for installer.jdownloader.org.
Here is the web.archive.org link: https://archive.org/details/jdownloader_202109
Do you want to use it instead of official one?

Reply:
Url replaced with archive.org one